conf/sample.ini

```diff
@@ -1295,6 +1295,18 @@ max_age =
 # Configures max number of alert annotations that Grafana stores. Default value is 0, which keeps all alert annotations.
 max_annotations_to_keep =
 
+[unified_alerting.recording_rules.writer]
```

Comment:
Alerting already has by far the most verbose section names in the entire grafana config. Can we drop the `.writer` suffix here? It seems to add clutter - there is no base recording rules section. And, recording rules by definition must write somewhere, it's effectively base functionality.

If we wanted to go a step further, consider:
- recording rules having their own section (`[recording_rules]` and that's it)
- putting these directly under the `[unified_alerting]` section, as we have config for many other subsystems there. For instance the redis HA config is there. (prefixing the key names accordingly)

Reply:
I don't know if we gain anything by prefixing the keys vs a separate subsection, but do see an argument for just `[recording_rules]`. Let's go with that 👍